sdk/cosmosdb/Azure.ResourceManager.CosmosDB/tests/ScenarioTests/CassandraResourcesOperationsTests.cs

```diff
@@ -0,0 +1,170 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core.TestFramework;
+using Azure.ResourceManager.CosmosDB.Models;
+using NUnit.Framework;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace Azure.ResourceManager.CosmosDB.Tests
+{
+    [TestFixture]
+    public class CassandraResourcesOperationsTests : CosmosDBManagementClientBase
+    {
+        public CassandraResourcesOperationsTests()
+            : base(true)
+        {
+        }
+
+        [SetUp]
+        public async Task ClearAndInitialize()
+        {
+            if (Mode == RecordedTestMode.Record || Mode == RecordedTestMode.Playback)
+            {
+                InitializeClients();
+                resourceGroupName = Recording.GenerateAssetName(CosmosDBTestUtilities.ResourceGroupPrefix);
+                await CosmosDBTestUtilities.TryRegisterResourceGroupAsync(ResourceGroupsOperations, CosmosDBTestUtilities.Location, resourceGroupName);
+            }
+        }
+
+        [TearDown]
+        public async Task CleanupResourceGroup()
+        {
+           await CleanupResourceGroupsAsync();
+        }
+
+        [TestCase]
+        public async Task CassandraCRUDTestsAsync()
+        {
+            var cosmosDBClient = GetCosmosDBManagementClient();
+
+            var locations = new List<Location>();
+            Location loc = new Location();
+            loc.LocationName = "West US";
+            loc.FailoverPriority = 0;
+            loc.IsZoneRedundant = false;
+            locations.Add(loc);
+
+            DatabaseAccountCreateUpdateParameters databaseAccountCreateUpdateParameters = new DatabaseAccountCreateUpdateParameters(locations);
+            databaseAccountCreateUpdateParameters.Location = "West US";
+            databaseAccountCreateUpdateParameters.Capabilities.Add(new Capability("EnableCassandra"));
+            databaseAccountCreateUpdateParameters.ConsistencyPolicy = new (DefaultConsistencyLevel.Eventual);
+
+            await WaitForCompletionAsync(await cosmosDBClient.DatabaseAccounts.StartCreateOrUpdateAsync(resourceGroupName, databaseAccountName, databaseAccountCreateUpdateParameters));
+
+            Task<Response> taskIsDatabaseNameExists = cosmosDBClient.DatabaseAccounts.CheckNameExistsAsync(databaseAccountName);
+            Response isDatabaseNameExists = taskIsDatabaseNameExists.ConfigureAwait(false).GetAwaiter().GetResult();
+            if (isDatabaseNameExists.Status != 200)
+            {
+                return;
```

Comment:
Why just return here? Isn't a non 200 a test failure? If so, you should: Assert.AreEqual(200, isDatabaseNameExists.Status)

Reply:
This is not resolved

Reply:
It is updated in the most recent commit, but I did not update in the comment.